src/blocks/card.css

```diff
@@ -0,0 +1,98 @@
+.container {
+  width: max-content;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.card__image {
+  margin-bottom: 40px;
+}
+
+.card__title {
+  width: 166px;
+  line-height: 18px;
+
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  color: #616070;
+  line-height: 14px;
+  width: 166px;
+  margin-bottom: 16px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
```

Comment:
Read about display:block elements

Reply:
what do you mean by that?
i believe flex fits best here